platform-includes/configuration/auto-session-tracking/javascript.node.mdx

```diff
@@ -14,17 +14,3 @@ Sentry.init({
   autoSessionTracking: false, // default: true
 });
 ```
-
-{/* TODO(v9): Remove this notice? */}
-
-<PlatformSection notSupported={["javacript.cordova", "javascript.capacitor", "javascript.electron"]}>
-
-<Alert title="Deprecation Notice">
-
-{/* We don't use platform links here because we always have to link to node and browser here and doing a conditional just for that feels overkill. */}
-
-Starting with SDK version 8.43.0 and up, the `autoSessionTracking` option has been deprecated. You can use the [BrowserSession integration](/platforms/javascript/configuration/integrations/browsersession/) in browser environments and the [Http integration](/platforms/javascript/guides/node/configuration/integrations/http/) (via the `trackIncomingRequestsAsSessions` option) in Node.js runtime.
```

Comment:
IIUC, `autoSessionTracking` is no longer available but we still mention it above. I'm not 100% familiar with the replacement but my guess is we should document the httpIntegration option mentioned in the deprecation notice?